Kickstarter-iOS/Views/Controllers/DiscoveryPageViewController.swift

```diff
@@ -11,6 +11,7 @@ internal final class DiscoveryPageViewController: UITableViewController {
 
   private var configUpdatedObserver: Any?
   private var currentEnvironmentChangedObserver: Any?
+  public var contentOffsetChanged: ((CGPoint) -> ())?
```

Comment:
Feel like we can use more closures, less delegates 🤔 I'll understand if there's resistance to this deviation at this stage and can probably update this to a delegate.

Reply:
Yeah I'd say we probably shouldn't break from convention for this one case 😅